File: packages/c#/Model/NitroPack.cs
namespace Com.CodeGame.CodeBall2018.DevKit.CSharpCgdk.Model
{
    public sealed class NitroPack : ISphere
    {
        public int id { get; set; }
        public double x { get; set; }
        public double y { get; set; }
        public double z { get; set; }
        public double radius { get; set; }
        public double nitro_amount { get; set; }
        public int? respawn_ticks { get; set; }
    }
}

File: packages/rust/src/model/nitro_pack.rs
#[derive(Debug, Serialize, Deserialize, Clone)]
pub struct NitroPack {
    pub id: i32,
    pub x: f64,
    pub y: f64,
    pub z: f64,
    pub radius: f64,
    pub nitro_amount: f64,
    pub respawn_ticks: Option<i32>,
}
